src/nodePty.ts

```diff
@@ -0,0 +1,22 @@
+import * as vscode from 'vscode'
```

Comment:
This file should get a small comment that at least point to the issue where you got the code snippet